ts/Series/Wordcloud/WordcloudSeries.ts

```diff
@@ -0,0 +1,691 @@
+/* *
+ *
+ *  Experimental Highcharts module which enables visualization of a word cloud.
+ *
+ *  (c) 2016-2020 Highsoft AS
+ *  Authors: Jon Arild Nygard
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ * */
+
+'use strict';
+
+import type BBoxObject from '../../Core/Renderer/BBoxObject';
+import type PositionObject from '../../Core/Renderer/PositionObject';
+import type SizeObject from '../../Core/Renderer/SizeObject';
+import type { StatesOptionsKey } from '../../Core/Series/StatesOptions';
+import type SVGAttributes from '../../Core/Renderer/SVG/SVGAttributes';
+import type SVGElement from '../../Core/Renderer/SVG/SVGElement';
+import type {
+    WordcloudSeriesOptions,
+    WordcloudSeriesRotationOptions
+} from './WordcloudSeriesOptions';
+import BaseSeries from '../../Core/Series/Series.js';
+import ColumnSeries from '../Column/ColumnSeries.js';
+import H from '../../Core/Globals.js';
+const { noop } = H;
+import LineSeries from '../Line/LineSeries.js';
+import PolygonMixin from '../../Mixins/Polygon.js';
+const {
+    getBoundingBoxFromPolygon,
+    getPolygon,
+    isPolygonsColliding,
+    rotate2DToOrigin,
+    rotate2DToPoint
+} = PolygonMixin;
+import U from '../../Core/Utilities.js';
+const {
+    extend,
+    find,
+    isArray,
+    isNumber,
+    isObject,
+    merge
+} = U;
+import WordcloudPoint from './WordcloudPoint.js';
+import WordcloudUtils from './WordcloudUtils.js';
+
+import '../Column/ColumnSeries.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
```

Comment:
We need to remove the namespace. So if these types are mainly used in the series class, move them to a `namespace WordcloudSeries` after the prototype properties. Search for `Class Namespace` to find examples.